### PR TITLE
Fix AC SA results ignoring AcLoadFlow outerloop status

### PR DESCRIPTION
**Pull request by jeandemanged**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

No

**What kind of change does this PR introduce?**

Bug fix

**What is the current behavior?**

SA results are taking into account NR status only

**What is the new behavior (if this is a feature change)?**
SA results are taking into account both OuterLoop status and NR status

**Does this PR introduce a breaking change or deprecate an API?**

No

**Other information**:
n/a

### CODE REVIEW

File: src/main/java/com/powsybl/openloadflow/sa/AbstractSecurityAnalysis.java [hunk 1]
```diff
@@ -89,23 +90,32 @@ public CompletableFuture<SecurityAnalysisReport> run(String workingVariantId, Se
...
 
-    public static LoadFlowResult.ComponentResult.Status loadFlowResultStatusFromNRStatus(NewtonRaphsonStatus status) {
-        return switch (status) {
-            case CONVERGED -> LoadFlowResult.ComponentResult.Status.CONVERGED;
-            case MAX_ITERATION_REACHED -> LoadFlowResult.ComponentResult.Status.MAX_ITERATION_REACHED;
-            case SOLVER_FAILED -> LoadFlowResult.ComponentResult.Status.SOLVER_FAILED;
-            case NO_CALCULATION, UNREALISTIC_STATE -> LoadFlowResult.ComponentResult.Status.FAILED;
-        };
+    public static LoadFlowResult.ComponentResult.Status componentResultStatusFromAcLoadFlowResult(AcLoadFlowResult result) {
+        // FIXME duplicates com.powsybl.openloadflow.OpenLoadFlowProvider.convertStatus
```
Comment (jeandemanged) [inline src/main/java/com/powsybl/openloadflow/sa/AbstractSecurityAnalysis.java:108]: @obrix @geofjamg would it be OK to just turn OpenLoadFlowProvider.convertStatus public ?

Comment (geofjamg) [reply src/main/java/com/powsybl/openloadflow/sa/AbstractSecurityAnalysis.java:108]: I would move it to AcLoadFlowResult as a static utility method.

Comment (jeandemanged) [reply src/main/java/com/powsybl/openloadflow/sa/AbstractSecurityAnalysis.java]: thanks, done

**Review by geofjamg — APPROVED** [review]

**Review by obrix — APPROVED** [review]
Looks good to me.

**Pull request merged**